### DISCUSSION TITLE
Roadmap to version 1.0.0

### DISCUSSION THREAD

**Discussion by dprada** [uibcdf/OpenPharmacophore#11] [Ideas]
This space can be used to discuss what are the features we want to add to OpenPharmacophore 1.0.0. In addition to this discussion, there is a [wiki page](https://github.com/uibcdf/OpenPharmacophore/wiki/Roadmap-to-version-1.0.0) where the list of goals should be exposed in a clean and sorted way for the future. But before, we can have an informal dialogue about what we are going to move to the wiki page.

**Comment by dprada** [comment]
# Pharmacophoric points definition.

We could start from a ligand-based analysis.

**Comment by dprada** [comment]
# Visualization of pharmacophoric points in NGLView.

Maybe this is not urgent. But it can be fun. Should this be done at the same time as implementing the pharmacophoric points? or later?

**Reply by Daniel-Ibarrola** [reply]
I think it should be implemented at the same time because it will be easier to compare our results with those obtained in the literature

**Reply by dprada** [reply]
Ok, it is a good idea.

**Comment by dprada** [comment]
# Pharmacophore model definition (agnostic -no matter how it was created-)

This goes obviously after having the pharmacophoric points. There is no need to agree on what should be the way to get the model. Having a class 'pharmacophore' to work with would be the first step.

**Comment by dprada** [comment]
# Pharmacophore models trajectories from molecular dynamics.

At a certain point, there must be a method to extract models from trajectories. Whether or not we have to include here strategies to sample the conformational space to get "better" models with MD, it can be a posterior goal to be achieved.

**Comment by dprada** [comment]
# Working with pharmacophore models to get consensus models or "representative" models.

Probably many approaches can be taken to do this. We should start with some of them, the easy and most standard ones.

**Comment by dprada** [comment]
# Markov State Models of pharmacophores to determine the most interesting models to be used for screening.

This has to be in 1.0.0. It should be one of the features that make this library different from other open source software.

**Comment by dprada** [comment]
# Screening chemical compounds with a pharmacophore model.

Here we can start with any open database, something like ZINCPharmer. Is there a Python API for ZINC? What other databases can we play with?

**Reply by Daniel-Ibarrola** [reply]
We can also try the [database of useful decoys](http://dude.docking.org/).

**Comment by dprada** [comment]
# Validation of the workflow

At this moment we have the desire of having tools to validate and test any workflow or approach to get and use pharmacophore models (based on OpenPharmacophore, in principle). But how to do this? is a bit blurry now... That's why there is a discussion opened as #10.

**Comment by dprada** [comment]
# Quality or confidence estimation of virtual screening results

This for sure has to be discussed and included before 1.0.0.

**Comment by dprada** [comment]
# Enhance sampling strategies to harvest the statistically relevant models avoiding brute force simulations.

I am not sure if we should include workflows to enhance the pharmacophoric conformational space with MD (adaptive sampling, etc). Probably yes, with OpenMM, but I am not really sure about this.

**Comment by dprada** [comment]
# First CI strategies implementation

In what version testing and CI strategies need to be implemented?
